extensions/sdk/src/__tests__/Rally.test.ts

```diff
@@ -143,7 +143,7 @@ describe("Rally SDK", function () {
 
     await invokeAuthChangedCallback(rally, null);
 
-    expect(browser.tabs.query).toBeCalledTimes(1);
+    expect(browser.tabs.query).toBeCalledTimes(2);
```

Comment:
why did this go up to 2? I don't see the code change causing that in this PR but maybe I'm missing something.